cmds/exp/acpigrep/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
2020

Reply:
maybe 2019-2020? this is an unrevert and modify of earlier 2019 work.

Reply:
ah nvm gotcha. done.